### DISCUSSION TITLE
Challenges, number of plots and the date all over the place

### DISCUSSION THREAD

**Discussion by kibirai** [Chia-Network/chia-blockchain#7544] [Q&A]
Hello, I'm farming solo since the beginning with my 625 plots today. Never had a problem, wallet and node are synced all the time. Since I have joined a pool, few days back, the challenges, the number of plots and the time when the plots are passing filter, everything is all over the place. I don't have even single one plot passing the filter corresponding with my time.
Before joining a pool, my solo plots was passing filters every ~10s corresponding to my date ant time and now it is a complete nonsense.

If I see that, my 625 solo plots are doing nothing?

I'm using v.1.2.1, everything is synchronized. I have 44 plots (for now) on the pool and 625 plots solo (without changing anything).

https://i.ibb.co/D5Lgyyt/chia-kib.jpg

Can someone explain this to me, please.
Thank You.

**Comment by Aidan24** [comment]
This happens to me too. It seems to only happen if I start plotting with the GUI (on version 1.2.0 onwards). If I kill the GUI after starting plotting (so the plotting processes run detached from the GUI) and then re-run the GUI, everything stays up to date. Although you now can't see the progress of the plots (without looking at the plot log files).

I also noticed from version 1.2.0 the progress bar has changed to show 100% before the plot is finished (and also the number of plots doesn't update correctly) so maybe something in how the plots are monitored has changed that is causing a resource leak or other issue.

**Reply by kibirai** [reply]
Good to know it isn't only me. I stopped farming on the pool, just disconnected the hdd with those plots. So now doing only solo farming again. I also restarted the GUI, everything is back to normal, but the problem is intermittent. All the info showing in GUI (number of plots, challenges, time) is messed up. Later, after few hours it back again to normal. And back again to a mess...

https://i.ibb.co/DVFfSd0/chia02.jpg

Also my config file, I have it set to INFO and now is full of "same" error. Never had this problem on v1.1.7. My log always was clean of nonsense messages.

https://i.ibb.co/YyzbHTk/chia03.jpg

Progress bar also showing info different comparing with v1.1.7. Now from 0% to 9% it takes maybe 3-4min and on v1.1.7 from 0% to 9% it would take 30min. You can think is doing faster now, but once you check the times on plot log files, they are the same (for me).
